package.json

```diff
@@ -53,7 +53,7 @@
     "@artsy/express-reloadable": "^1.3.1",
     "@artsy/palette": "^2.18.1",
     "@artsy/passport": "^1.1.0",
-    "@artsy/reaction": "^5.11.0",
+    "@artsy/reaction": "^6.0.0",
```

Comment:
I wonder if this version upgrade should separately be deployed to be safe.

cc @alloy

Reply:
Yeah sounds like it is happening separately, this is on hold to wait until the upgrade is merged.